tensorboard/webapp/widgets/card_fob/card_fob_controller_test.ts

```diff
@@ -94,6 +96,7 @@ describe('card_fob_controller', () => {
   function createComponent(input: {
     axisDirection?: AxisDirection;
     timeSelection: TimeSelection;
+    enableRangeSelection: boolean;
```

Comment:
I think this should be optional and default to false when not specified.

Reply:
That's reasonable.